delighted/resource.py

```diff
@@ -150,6 +150,37 @@ class Bounce(AllResource):
     path = 'bounces'
 
 
+class AutopilotConfiguration(RetrievableResource):
+    path = 'autopilot'
+
+
+class AutopilotMembership:
+
+    @classmethod
+    def forEmail(self):
+        return AutopilotMembershipForEmail
+
+    @classmethod
+    def forSms(self):
+        return AutopilotMembershipForSms
+
+    @classmethod
+    def delete(self, **params):
+        try:
+            self._set_client(params)
+            return self.client.request_json('delete', self.path, {}, params)
+        except AttributeError:
+            print('You must first call a platform-specific method (e.g. forEmail())')
```

Comment:
do you want to re-raise the exception here so that it doesn't look like the call succeeded?